packages/amplify-e2e-tests/src/api-directives/tests/auth-fieldLevelAuth5.ts

```diff
@@ -0,0 +1,66 @@
+//schema
+export const schema = `
+##error: missing the owner field, 
+##change: added the missign owner field
+##change: removed the  on the updateAt field // or @auth(rules: [{ allow: groups, groups: ["Admins"] }])
+
+type Todo 
+  @model
+{
+  id: ID! 
+  owner: String
+  updatedAt: AWSDateTime! @auth(rules: [{ allow: owner, operations: [update] }])
+  content: String! @auth(rules: [{ allow: owner, operations: [update] }])
+}
+
+##fieldLevelAuth5`;
+//mutations
+export const mutation1 = `
+mutation CreateTodo {
+    createTodo(input: {
+      id: "1",
+      owner: "user1",
+      updatedAt: "2020-01-01T01:05:49.129Z"
+      content: "todo1 content"
+    }) {
+      id
+      owner
+      updatedAt
+      content
+    }
+}`;
+export const expected_result_mutation1 = {
```

Comment:
Please dont use the variable naming magic. Like i suggested in an earlier comment, move this inside a test and then wrap it inside an object, so both mutation and results are wrapped in the same object